mkdocs/docs/HPC/connecting.md

```diff
@@ -759,3 +759,17 @@ created.
 ### Fast file transfer for large datasets
 
 See [the section on `rsync` in chapter 5 of the Linux intro manual](../linux-tutorial/uploading_files/#copying-faster-with-rsync).
+
+## Changing login nodes
+It can be useful to work on the same login node using multiple terminal sessions. However, when you connect to the HPC (High-Performance Computing) system through SSH, you're directed to a random login node, which might not be the one where you already have an active session. To address this, there is a way to manually switch your active login node.
+
+For instance, if you want to switch to the login node named `gligar07`, you can use the following command while you are connected to the HPC:
+```shell
+ssh gligar07
```

Comment:
Mention that this makes sense when you're logged in to `gligar08`, and how to check where you're logged in (run the `hostname` command, show output of hostname before and after using `ssh`).

And also mention the reverse (use `ssh gligar08` if you got logged in to `gligar07` but want to be on `gligar08`).